lzy/allocator/src/test/java/ai/lzy/allocator/test/AllocatorServiceMountsTest.java

```diff
@@ -467,6 +458,47 @@ public void vmShouldNotAllocateMountPodInSystemCluster() throws Exception {
         Assert.assertNull(mountPodName);
     }
 
+    @Test
+    public void unmountShouldRecreateMountPodWithNotMountedDisk() throws Exception {
+        var sessionId = createSession(Durations.fromDays(10));
+        var allocatedVm = allocateVm(sessionId);
+        var vm = vmDao.get(allocatedVm.vmId(), null);
+        Assert.assertNotNull(vm);
+        var dynamicMount = createReadyMount(vm, allocatedVm.allocationOpId());
+        var update = DynamicMount.Update.builder()
+            .mounted(false)
+            .build();
+        var mountPodName = "foobar-42";
+        vmDao.setMountPodAndIncrementNextId(vm.vmId(), mountPodName, null);
+        dynamicMount = dynamicMountDao.update(dynamicMount.id(), update, null);
+        Assert.assertNotNull(dynamicMount);
+
+        var mountPodCreated = new AtomicBoolean(false);
+        var otherPodsDeleted = new AtomicBoolean(false);
+        var volumeDeleted = new AtomicBoolean(false);
+        var volumeClaimDeleted = new AtomicBoolean(false);
+        mockCreatePod(pod -> mountPodCreated.set(true));
+        mockUnmountCall();
+        mockDeletePods(200, () -> otherPodsDeleted.set(true));
```

Comment:
nit: it'd be better to replace plain literal `200` with constant `HttpURLConnection.HTTP_OK`